src/LuaMultiThreaded2.cpp

```diff
@@ -68,6 +69,96 @@ extern "C" int errorHandler(LuaState * L)
 
 
 
+extern "C" static int mutexNew(LuaState * aState)
```

Comment:
Please add a doxy-comment to all functions, especially those that provide the Lua API.